patches/tModLoader/Terraria/ModLoader/AutoloadAttribute.cs

```diff
@@ -8,19 +8,26 @@ namespace Terraria.ModLoader
 	/// True to always autoload, false to never autoload, null to use mod default.
 	/// </summary>
 	[AttributeUsage(AttributeTargets.Class | AttributeTargets.Interface, AllowMultiple = true, Inherited = true)]
-	public class AutoloadAttribute : Attribute
+	public sealed class AutoloadAttribute : Attribute
```

Comment:
Are we sure we want this sealed?

Reply:
99%, it's an attribute, it doesn't make sense to inherit from

Reply:
What if you want some custom logic to follow the auto loading?